views/sold-land-filter.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Has the entity disposed of UK property or land since 28 February 2022?
+ - {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include "includes/list/errors.html" %}
+
+      <form method="post">
+
+
+        {{ govukRadios({
+          errorMessage: errors.has_sold_land if errors,
+          classes: "govuk-radios--inline",
+          idPrefix: "has_sold_land",
+          name: "has_sold_land",
+          fieldset: {
+            legend: {
+              text: "Has the entity disposed of UK property or land since 28 February 2022?",
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--xl"
+            }
+          },
+          items: [
+            {
+              value: 1,
+              text: "Yes",
+              checked: has_sold_land == 1
+            },
+            {
+              value: 0,
+              text: "No",
+              checked: has_sold_land == 0
```

Comment:
We need to return the value in the `render` call if we want to visualise what has been selected

Reply:
We can leave it, the logic will be completed when we store the info into the session and returned on the `render`